jenkinsci/redgate-sql-ci-plugin
# Add data compare options

Added a data compare options field to the configuration for Jenkins

Why?
We have the data compare options supported by the PowerShell cmdlets which can now be used by users of Jenkins.

Who worked with you?
@IvoMiller @mayamalakova

Automated testing
N/A

Manual testing
  Check Jenkins build logs contain specified data compare options for latest version.
  Check that invalid options are handled correctly
  Check that warning is displayed for older version of SCA when trying to use data compare options.
  Check that multiple data compare options are handled and passed correctly.
Release Notes and Documentation
Update release notes.

Architectural Decision Record
N/A

Risk Assessment
Not very big. Slight tweaks to SqlCi.ps1 to handle optionally adding new paraemter according to version most likely source of error. Changes have been thoroughly tested manually.
